cs/di-defaultextensions.texy

```diff
@@ -0,0 +1,105 @@
+DI: Základní rozšíření pro kontejner
+************************************
+
+Nette obsahuje několik předdefinovaných [Nette\DI\CompilerExtension |api:] pro DI kontejner.
+
+ExtensionsExtension
+===================
+
+Díky tomuto rozšíření můžeme zaregistrovat další rozšiření pomocí konfiguračních NEON souborů.
+
+/---neon
+extensions:
+	dibi: Dibi\Bridges\Nette\DibiExtension22
+\---
+
+Tímto způsobem přidáme rozšíření `Dibi\Bridges\Nette\DibiExtension22` pod názvem *dibi*, takže následně ho konfigurujeme pod sekcí dibi:
+
+/---neon
+dibi:
+	host: localhost
+\---
+
+Můžeme také předat specifické parametry pro naše rozšíření.
```

Comment:
Je to dost lamane, taky jsem premyslel o tom, zda-li tam ma zaznit slovo konstruktor. :-)